test/DebugInfo/for.swift

```diff
@@ -7,7 +7,7 @@ for i in 0 ..< 3 {
 }
 
 for i in 0 ..< 3 {
-// CHECK-DAG: ![[SCOPE2:[0-9]+]] = {{.*}}Block(scope: ![[MAIN]],{{.*}}line: [[@LINE-1]]
+// CHECK-DAG: ![[SCOPE2:[0-9]+]] = {{.*}}Block(scope: ![[MAIN2:[0-9]+]],{{.*}}line: [[@LINE-1]]
```

Comment:
Same here.. why aren't they in the same parent scope `DS`?

Reply:
We resolved this offline.